assets/pages/new-contributions-landing/components/ContributionThankYou.jsx

```diff
@@ -4,50 +4,66 @@
 
 import React from 'react';
 import { connect } from 'react-redux';
-
 import { type Contrib, getSpokenType } from 'helpers/contributions';
 import { classNameWithModifiers } from 'helpers/utilities';
-
-import SvgArrowRight from 'components/svgs/arrowRightStraight';
-import MarketingConsent from '../components/marketingConsent';
+import MarketingConsent from '../components/MarketingConsent';
+import { ButtonWithRightArrow } from '../components/ButtonWithRightArrow';
 
 // ----- Types ----- //
 
 /* eslint-disable react/no-unused-prop-types */
 type PropTypes = {
   contributionType: Contrib,
+  // wil be true when this is the first thank you page a Direct Debit recurring contributor sees
```

Comment:
will

Reply:
I'm still struggling to how this logic works. where's the bit that decides it's the first thank you page seen? I only see

```jsx
thankYou: (<ContributionThankYou showDirectDebitCopy={props.paymentMethod === 'DirectDebit'} />),
```